backend/layers/business/business.py

```diff
@@ -297,34 +350,54 @@ def update_collection_version(
         # Check metadata
         validation.verify_collection_metadata_update(body, errors)
 
-        current_version = self.get_collection_version(version_id)
-        if current_version.published_at is not None:
+        current_collection_version = self.get_collection_version(version_id)
+        if current_collection_version.published_at is not None:
             raise CollectionUpdateException(["Cannot update a published collection"])
 
         # Determine if publisher metadata should be unset, ignored or set at the end of the method.
         # Note: the update needs to be done at the end to ensure atomicity
         unset_publisher_metadata = False
         publisher_metadata_to_set = None
 
+        new_doi = None
+        current_doi = None
         if not ignore_doi_update:
```

Comment:
I realize you didn't commit this, but can you change this to a positive-usage variable `update_doi`. It makes more sense given the logic of the usage. Trying to avoid `if not`s. @Bento007 will agree

Reply:
I will handle this in a separate PR to avoid increasing this PR's scope